src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -633,7 +633,13 @@ impl ThinPool {
         }
 
         let overall_used_pct = used_pct(*used, *used + *available);
-        info!("Data tier percent used: {}", overall_used_pct);
+        debug!("Data tier percent used: {}", overall_used_pct);
+        if overall_used_pct >= 90 {
```

Comment:
There is a constant you can use here instead of a magic number `const SPACE_WARN_PCT: u8 = 90;`.  Also according to the constants themselves, critical is defined as >= 95% `SPACE_CRIT_PCT` so I'm wondering if the info statement should be slightly reworded.

Reply:
Also instead of introducing a new conditional check, we could insert the info message into the existing conditional assignment, something to consider.

```rust
        let new_state = if overall_used_pct < SPACE_WARN_PCT {
            FreeSpaceState::Good
        } else if overall_used_pct < SPACE_CRIT_PCT {
            FreeSpaceState::Warn
        } else {
             info!(
                "Percent used has reached a critical threshold level; data tier percent used: {}",
                overall_used_pct
            );
            FreeSpaceState::Crit
        };
```